src/Documentation/FixerDocumentGenerator.php

```diff
@@ -89,6 +90,19 @@ public function generateFixerDocumentation(FixerInterface $fixer): string
             }
         }
 
+        $experimentalDescription = '';
+
+        if ($fixer instanceof ExperimentalFixerInterface) {
+            $experimentalDescriptionRaw = RstUtils::toRst('It is not covered with backward compatibility promise and may produce unstable or unexpected results.', 0);
```

Comment:
For me this description is not clear enough. There's no clear distinction between _experimental_ and _risky_ rules - both may produce unexpected results. I would rather focus on:
- rule/options naming that may be changed in minor releases
- options may be also removed
- default config may be changed

It's not about riskiness, but about the contract between us and the users. It must be clear that API of the experimental rules may be changed at any point until it's considered stable or is dropped completely.

Reply:
Then, if the rule is both experimental _and_ risky, we will document both the unstable API _and_ potentially unexpected output. But I believe there may be rules that are _not_ risky (like `numeric_literal_separator`) but are experimental because of undecided defaults.

Reply:
For now the content is simply moved around, from here: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7729/files#diff-4dcfa529cee9ae4b3e7334cea22c21d87465fa962dfd3badf31d05f62c123a5cL87

Would you like to propose new content?

Reply:
I would like to see something like this:

```suggestion
            $experimentalDescriptionRaw = RstUtils::toRst('Rule is not covered with backward compatibility promise, use it at your own risk. Rule\'s behaviour may be changed at any point, including rule\'s name; its option\'s names, availability and allowed values; its default configuration. Rule may be even removed without prior notice. Feel free to provide feedback and help with determining final state of the rule.', 0);
```